opensquirrel/passes/exporter/cqasmv1_exporter.py

```diff
@@ -14,12 +15,11 @@ class _CQASMv1Creator(IRVisitor):
     # Precision used when writing out a float number
     FLOAT_PRECISION = 8
 
-    def __init__(self, register_manager: RegisterManager) -> None:
+    def __init__(self, register_manager: RegisterManager, barrier_links: list[list[Statement]]) -> None:
         self.register_manager = register_manager
         qubit_register_size = self.register_manager.get_qubit_register_size()
-        self.cqasmv1_string = "version 1.0\n\n{}\n\n".format(
-            f"qubits {qubit_register_size}" if qubit_register_size > 0 else ""
-        )
+        self.output = "version 1.0\n\n{}\n\n".format(f"qubits {qubit_register_size}" if qubit_register_size > 0 else "")
+        self.barrier_links = barrier_links
```

Comment:
Where does this 'links' terminology come from? In the code, we talk about "groups of 'linked' barriers". But where do we explain what are these `barrier_links`?

Reply:
Good point! We should indeed be consistent in our terms. They are indeed groups of linked barriers.